### PR TITLE
Dashboards: Add option to specify explicit percent change text size for stat panels

### PR DESCRIPTION
**Pull request by XZCendence**
**What is this feature?**

This feature adds an option in the "Text size" category for stat panels to set the explicit text size of the percent change markers, and the arrow icon along with it.

**Why do we need this feature?**

On smaller or nonstandard displays with certain aspect ratios/resolutions, the percent change marker can appear too small or too large. Currently, only the title and value text sizes are customizable on stat panels. This feature completes the customization by allowing users to explicitly control the text size of percent change markers.

**Who is this feature for?**

This feature is for any and all Grafanistas who want to be able to fully customize the text size of everything on their stat panels. This becomes especially relevant when using grafana-image-renderer to render out a stat panel solo. 

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

**Comment by XZCendence** [comment]
Example of how this looks in action (in this example the percent change would normally be too big)
image

**Review by mckn — COMMENTED** [review]
LGTM! But I'm letting one of the code owners to do the final review.

File: packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx [hunk 1]
```diff
@@ -111,8 +116,8 @@ export abstract class BigValueLayout {
   ): PercentChangeStyles {
     const VALUE_TO_PERCENT_CHANGE_RATIO = 2.5;
     const valueContainerStyles = this.getValueAndTitleContainerStyles();
-    const percentFontSize = Math.max(this.valueFontSize / VALUE_TO_PERCENT_CHANGE_RATIO, 12);
-    let iconSize = Math.max(this.valueFontSize / 3, 10);
+    const percentFontSize = this.percentFontSize || Math.max(this.valueFontSize / VALUE_TO_PERCENT_CHANGE_RATIO, 12);
+    let iconSize = this.percentFontSize ? this.percentFontSize - 3 : Math.max(this.valueFontSize / 3, 10);
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx:120]: The `this.percentFontSize - 3` seems quite arbitrary, will it work with any font size?

Comment (XZCendence) [reply packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx:120]: Yes, I did that to ensure the size of the icon is consistent with the regular auto-calculated icon size (the behavior you got before I made any changes, or when leaving the font size empty or 0)

**Review by Clarity-89 — APPROVED** [review]

**Comment by Clarity-89** [comment]
@XZCendence you'll need to update your branch to the latest main.

**Comment by XZCendence** [comment]
Just synced it now

**Comment by Clarity-89** [comment]
@XZCendence sorry for the delay, we had to make some fixes to our CI pipelines for external contributors, could you update your branch to the latest main once more?

**Comment by XZCendence** [comment]
@Clarity-89 Just synced it.

**Pull request merged**